definitions/pipelines/immuno.cwl

```diff
@@ -0,0 +1,777 @@
+    cle_vcf_filter:
+        type: boolean
+        default: false
+    variants_to_table_fields:
+        type: string[]
+        default: [CHROM,POS,ID,REF,ALT,set,AC,AF]
+    variants_to_table_genotype_fields:
+        type: string[]
+        default: [GT,AD]
+    vep_to_table_fields:
+        type: string[]
+        default: [HGVSc,HGVSp]
+    custom_gnomad_vcf:
+        type: File?
+        secondaryFiles: [.tbi]
+    custom_clinvar_vcf:
+        type: File?
+        secondaryFiles: [.tbi]
+    vep_assembly:
+        type: string
+        doc: Used to explicitly define which assembly to use; required if there are two or more in the same directory
+    manta_call_regions:
+        type: File?
+        secondaryFiles: [.tbi]
+    manta_non_wgs:
+        type: boolean?
+        default: true
+    manta_output_contigs:
+        type: boolean?
+    somalier_vcf:
+        type: File
+
+    #germline inputs
+    emit_reference_confidence:
+        type: string
+    gvcf_gq_bands:
+        type: string[]
+    gatk_haplotypecaller_intervals:
+        type:
+            type: array
+            items:
+                type: array
+                items: string
+    optitype_name:
+        type: string?
+
+    #phase_vcf inputs
+    reference_dict:
+        type: File?
+
+    #pvacseq inputs
+    readcount_minimum_base_quality:
+        type: int?
+    readcount_minimum_mapping_quality:
+        type: int?
+    prediction_algorithms:
+        type: string[]
+    epitope_lengths:
+        type: int[]?
+    binding_threshold:
+        type: int?
+    allele_specific_binding_thresholds:
+        type: boolean?
+    minimum_fold_change:
+        type: float?
+    peptide_sequence_length:
+        type: int?
+    top_score_metric:
+        type:
+            - "null"
+            - type: enum
+              symbols: ["lowest", "median"]
+    additional_report_columns:
+        type:
+            - "null"
+            - type: enum
+              symbols: ["sample_name"]
+    fasta_size:
+        type: int?
+    downstream_sequence_length:
+        type: string?
+    exclude_nas:
+        type: boolean?
+    phased_proximal_variants_vcf:
+        type: File?
+        secondaryFiles: ['.tbi']
+    maximum_transcript_support_level:
+        type:
+            - "null"
+            - type: enum
+              symbols: ["1", "2", "3", "4", "5"]
+    normal_cov:
+        type: int?
+    tdna_cov:
+        type: int?
+    trna_cov:
+        type: int?
+    normal_vaf:
+        type: float?
+    tdna_vaf:
+        type: float?
+    trna_vaf:
+        type: float?
+    expn_val:
+        type: float?
+    net_chop_method:
+        type:
+            - "null"
+            - type: enum
+              symbols: ["cterm", "20s"]
+    net_chop_threshold:
+        type: float?
+    netmhc_stab:
+        type: boolean?
+    pvacseq_threads:
+        type: int?
```

Comment:
Is there a reason to expose this?  In general, our practice has seemed to be to set appropriate resource requests in the CWL for number of threads and memory.

Reply:
We're still trying to figure out what those numbers should be. Until we do I think it's ok to expose this to give us the flexibility to easily change this parameter.